ci/auto/local-postgres15.env

```diff
@@ -14,3 +14,5 @@ TYK_PMP_PUMPS_SQLAGGREGATE_TYPE=sql_aggregate
 TYK_PMP_PUMPS_SQLAGGREGATE_META_TYPE=postgres
 TYK_PMP_PUMPS_SQLAGGREGATE_META_CONNECTIONSTRING="host=postgres port=5432 user=postgres dbname=tyk_analytics"
 TYK_PMP_PUMPS_SQLAGGREGATE_META_TABLESHARDING=true
+TYK_GW_SLAVEOPTIONS_RPCKEY=placeholder
+TYK_GW_SLAVEOPTIONS_APIKEY=placeholder
```

Comment:
This variables shouldn't be here, we need to find another place that makes this sustainable.